src/resources/js/v2/tickets-block.js

```diff
@@ -183,9 +183,10 @@ tribe.tickets.block = {
 			}
 
 			const $price = $input.closest( obj.selectors.item ).find( obj.selectors.itemPrice ).first();
+			const real_price = $input.closest( obj.selectors.item ).data( 'ticket-price' );
 			let quantity = parseInt( $input.val(), 10 );
 			quantity = isNaN( quantity ) ? 0 : quantity;
-			let text = $price.text();
+			let text = isNaN( real_price ) ? $price.text() : real_price.toString();
 			text = tribe.tickets.utils.cleanNumber( text, provider );
```

Comment:
I know this is existing, but let's use a different variable here for the first `text` variable and use `const` rather than overwriting the `text` variable again.